scripts/create-secrets/prompts/inquirer-fuzzy-path.js

```diff
@@ -0,0 +1,100 @@
+const fs = require("fs");
+const path = require("path");
+const util = require("util");
+
+const Choices = require("inquirer/lib/objects/choices");
+const InquirerAutocomplete = require("inquirer-autocomplete-prompt");
+const stripAnsi = require("strip-ansi");
+const style = require("ansi-styles");
+const fuzzy = require("fuzzy");
+
+const readdir_ = util.promisify(fs.readdir);
+
+class InquirerFuzzyPath extends InquirerAutocomplete {
```

Comment:
Could we put some JsDoc on this? I'm not sure what `InquirerFuzzyPath` is supposed to do just from the name.

Reply:
Good idea~ Will put some description~ This class is a plugin to work as file picker.